htdocs/compta/resultat/result.php

```diff
@@ -354,7 +354,11 @@
 					//$r = $AccCat->calculate($result);
 					$r = (float) dol_eval($result, 1, 1, '1');
 
-					print '<td class="liste_total right"><span class="amount">'.price($r).'</span></td>';
```

Comment:
No need for if !empty when usin getDol...
Use if getDol...

Reply:
Done